libgoal/lockedFileUnix.go

```diff
@@ -14,64 +14,60 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
 
-// +build !linux,!windows
+// Support all unix system except linux
+// in https://github.com/golang/sys/blob/master/unix/syscall_unix.go
+
+// +build aix darwin dragonfly freebsd netbsd openbsd solaris
 
 package libgoal
 
 import (
+	"golang.org/x/sys/unix"
```

Comment:
nit: same as before, move it to be in it's own group after the built-ins group.

Reply:
fixed